lib/db.js

```diff
@@ -48,6 +49,7 @@ module.exports = (
         return account
       })
   }
+  const getGeoData = geodb(log)
```

Comment:
can we move `const getGeoData = geodb(log)` above the function, and put it with the rest of the `const` declarations?